packages/paste-core/primitives/box/package.json

```diff
@@ -49,6 +49,7 @@
     "@twilio-paste/types": "^3.1.8",
     "prop-types": "^15.7.2",
     "react": "^17.0.2",
-    "react-dom": "^17.0.2"
+    "react-dom": "^17.0.2",
+    "typescript": "^4.6.4"
```

Comment:
Note: we should add this everywhere so we can build directly from within a single package. That's why I added it here. Won't require a changeset because it's just for us.